lib/puppetserver/ca/action/generate.rb

```diff
@@ -140,7 +144,17 @@ def run(input)
           # Generate and save certs and associated keys
           if input['ca-client']
             # Refused to generate certs offfline if the CA service is running
-            return 1 if HttpClient.check_server_online(puppet.settings, @logger)
+            begin
+              if HttpClient.check_server_online(puppet.settings, @logger, input['force'])
+                return 1
+              else
+                @logger.inform("Continuing with certificate signing.") if input['force']
```

Comment:
Does this mean we will proceed even if the server is definitely online (`check_server_online` returned false)? I think that's not what we wanted...